pkg/core/basic_cluster.go

```diff
@@ -137,8 +137,10 @@ type StoreSetInformer interface {
 
 // StoreSetController is used to control stores' status.
 type StoreSetController interface {
-	PauseLeaderTransfer(id uint64) error
-	ResumeLeaderTransfer(id uint64)
+	PauseLeaderTransferIn(id uint64) error
```

Comment:
How about adding a parameter?

Reply:
Is it something like `PauseLeaderTransferIn(id uint64, d direction) error`

Reply:
yes, WDYT?